spec/factories/games.rb

```diff
@@ -1,5 +1,7 @@
 FactoryGirl.define do
   factory :game do
     game_name { Faker::Lorem.sentence }
+    association :white_user, factory: :user
+    association :black_user, factory: :user
```

Comment:
updated factory to now add a white and black user when the game factory is called.